### DISCUSSION TITLE
Code Compare: Add option for {"diffEditor.ignoreTrimWhitespace": true}

### DISCUSSION THREAD

**Discussion by MikeWork123** [arnoudkooi/SN-Utils#456] [Ideas]
Is it possible in the code compare to add an option to ignore white space and possibly line feed changes to make it easier to compare large scripts that having spacing differences you don't care about?

**Comment by arnoudkooi** [comment]
I believe that is the default setting, when I test I get this result. I think you would want to ignore this:

![Google Chrome20231219 - 1730](https://github.com/arnoudkooi/ServiceNow-Utils/assets/840781/6d3ea4fe-a23b-41c1-bdde-a66d430d512e)
Looking at [this issue](https://github.com/microsoft/monaco-editor/issues/360) it seems this is not supported
